modules/shared/src/log-metric.ts

```diff
@@ -0,0 +1,71 @@
+import debug = require('debug');
+import fetch from 'node-fetch';
+
+/**
+ * Logs a metric to a remote logging server, defined by the environment vars:
+ * - `BUGBOT_LOG_METRICS_URL`
+ * - `BUGBOT_LOG_METRICS_AUTH`
+ *
+ * This code assumes that it is sending logs to an instance of Grafana Loki,
+ * with the data format defined by:
+ * https://grafana.com/docs/loki/latest/api/#post-lokiapiv1push
+ */
+export function logMetric(
+  message: string,
+  labels?: Record<string, string>,
```

Comment:
```suggestion
  labels: Record<string, string> = {},
```

_opinion_ IMO the ` = {}` is slightly preferable here to avoid code like `...(labels || {})` below